src/modules/_nomodule/gl-context.js

```diff
@@ -9,7 +9,7 @@ module.exports = function runInBrowserContext(input, callback, step, options) {
 
     var obj = { input: input, modOptions: minOptions }
 
-    puppeteer.launch().then(function(browser) {
+    puppeteer.launch({headless: true, args:['--no-sandbox --disable-setuid-sandbox']}).then(function(browser) {
```

Comment:
Tried a couple new things here. Maybe we should just merge this if it passes and then see if it works upstream? @VibhorCodecianGupta @HarshKhandeparkar @Divy123 ?

Reply:
Sounds good. I'd test this but I do not have a GCF instance. `image-sequencer-app` is deployed on GCF, right @jywarren ?

Reply:
@jywarren I also do not have GCF instance. Any suggestions on testing it?

Reply:
We can test it on some cloud service like Google Cloud or firebase just to make sure it isn't broken

Reply:
Thanks, i just added @Divy123 and @VibhorCodecianGupta -- we're using this in Cloud Functions:

https://gist.github.com/jywarren/7721a38725affdbc4fad2598ff1be59e

@HarshKhandeparkar i can add you too if you want to send me your email address. 

Thanks!

Reply:
More info here: https://github.com/publiclab/image-sequencer-app/issues/6

Thanks, all!

Reply:
@jywarren amazing, I'll try running it.
 
Also, I think issues #1030 #1033 are pretty urgent, I listed some possible fixes and your review is required there. Plugging in my proposal updates to your response as well (hope that's fine! :D) https://bit.ly/2VlDshL